src/Session.h

```diff
@@ -236,6 +236,14 @@ class Session {
 
     // Close cached image if it has been updated
     void CloseCachedImage(const std::string& directory, const std::string& file);
+    bool AnimationActive() {
+        return _animation_active;
+    }
+    void SetAnimationActive(bool val) {
+        fprintf(stderr, " SetAniActive %d\n", val);
```

Comment:
please use `spdlog::debug` for this

Reply:
Please also use the full function name.

(Unless this was meant to be temporary and should just be removed.)